style.css

```diff
@@ -36,6 +36,10 @@ body{
 .header{
     background-color: #ffffff;
     border-bottom: 1px solid #e2e8f0;
+    position: fixed;
```

Comment:
need some top margin to the main section

Reply:
added top margin to the content div 
![Screenshot 2024-09-25 213752](https://github.com/user-attachments/assets/e28095e4-e0e0-4a7b-a3d2-bdbe88f2a9ed)